README.md

```diff
@@ -53,20 +53,6 @@ $ python main.py
 
 <br>
```

Comment:
Remove the `<br>` here, you already added one in the previous text line which leads into a bit bigger empty space.

Reply:
![image](https://user-images.githubusercontent.com/68811721/198871313-65975833-b66f-4758-bf8a-0aaeb971aea1.png)

Reply:
@Omar8345 Done!